File: jour_8/jour8.c
#include<stdio.h>
int annee;
int main(){
    printf("Entrez l'année : ");
    scanf("%d", &annee);
    printf(annee % 4 == 0? annee % 100 == 0 ? annee % 400 == 0 ? "cette année est bisextille\n" : "cette année n'est pas bisextille\n" : "cette année est bixectille\n" : "cette année n est pas bisextille \n");
    return 0;
}

File: jour_8/jour8.pas
program VerifBissextile;
var
    annee: Integer;

begin
    write('Entrez l''annee : ');
    readln(annee);
    if ((annee mod 4 = 0) and (annee mod 100 <> 0)) or (annee mod 400 = 0) then
    writeln(annee, ' est une annee bissextile.')
    else
        writeln(annee, ' n''est pas une annee bissextile.');
    
    readln; 
end.